pkg/types/azure/defaults/machines.go

```diff
@@ -17,7 +17,7 @@ func BootstrapInstanceType(region string) string {
 // D4s v3 gives us 4 CPU's, 16GiB ram and 32GiB of temporary storage
```

Comment:
update comment

Reply:
also would like to see the docs updated here https://github.com/openshift/installer/blob/master/docs/user/azure/limits.md#instance-limits